Realm/Realm.Weaver/Extensions/ModuleDefinitionExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
```

Comment:
Should this be the year the file is created (2021) or a fixed 2016?

Reply:
Yeah, that should be 2021.